metricbeat/docs/developer-guide/create-metricset.asciidoc

```diff
@@ -172,34 +168,35 @@ func (m *MetricSet) Fetch() (common.MapStr, error) {
 }
 ----
 
-Fetch must return a `common.MapStr` which will be translated to the JSON content.
+`Fetch` must return a `common.MapStr`, which will be translated to the JSON content.
 The JSON output will be identical to the naming and structure you use in
-common.MapStr. More details about MapStr and its functions can be found in the API
-docs https://godoc.org/github.com/elastic/beats/libbeat/common#MapStr[here].
+`common.MapStr`. For more details about `MapStr` and its functions, see the
+https://godoc.org/github.com/elastic/beats/libbeat/common#MapStr[MapStr API docs].
 
 
 [float]
 ==== Multi Fetching
-Metricbeat has two different Fetch interfaces. One is to fetch a single event
-which is shown above. In some cases every fetch returns a list of events. Instead
-of using an array inside a JSON document, we recommend to create a list of events.
+Metricbeat has two different `Fetch` interfaces. One of the interfaces, which you saw in the
+previous example, fetches a single event. In some cases, every fetch returns a list of events.
+Instead of using an array inside a JSON document, we recommend that you create a list of events.
 
-For this kind of data the following interface can be used:
+For this kind of data, you can use the following `Fetch` interface:
 
 [source,go]
 ----
 (m *MetricSet) Fetch() ([]common.MapStr, error)
 ----
 
-The only difference to the previous example is that a `[]common.MapStr` is returned.
+The only difference between this and the previous example is that the second example returns `[]common.MapStr`.
 Metricbeat will send all events with the same timestamp and a unique identifier
```

Comment:
Fixed...also reworded this sentence because it was grammatically ambiguous. The sentence now says: "Metricbeat will add the same timestamp to all the events in the list to make it possible to correlate the events."